nncf/common/tensor_statistics/aggregator.py

```diff
@@ -79,9 +79,14 @@ def collect_statistics(self, model: TModel, graph: NNCFGraph) -> None:
             outputs = engine.infer(input_data)
             processed_outputs = self._process_outputs(outputs)
             self._register_statistics(processed_outputs, merged_statistics)
-            empty_statistics = False
-        if empty_statistics:
+            processed_samples += 1
+        if processed_samples == 0:
             raise nncf.ValidationError(EMPTY_DATASET_ERROR)
+        if self.stat_subset_size is not None and self.stat_subset_size > processed_samples:
+            nncf_logger.warning(
+                f"Dataset contains only {processed_samples} samples, "
```

Comment:
@l-bat, @eaidova, @nikita-savelyevv please check that this is not impact NNCF examples, OpenVINO notebooks and optimum-intel.

Reply:
@zina-cs Could you please provide `subset_size` to the `nncf.quantize` in the following examples?
1. https://github.com/openvinotoolkit/nncf/tree/develop/examples/post_training_quantization/openvino: 
    * anomaly_stfpm_quantize_with_accuracy_control - `subset_size=109`
    * yolov8 - `subset_size=128`
    * yolov8_quantize_with_accuracy_control - `subset_size=128`
2. https://github.com/openvinotoolkit/nncf/tree/develop/examples/post_training_quantization/onnx/yolov8_quantize_with_accuracy_control - `subset_size=128`
3. https://github.com/openvinotoolkit/nncf/tree/develop/examples/post_training_quantization/torch/ssd300_vgg16 - `subset_size=128`